wolfcrypt/src/memory.c

```diff
@@ -406,7 +406,7 @@ int wolfSSL_load_static_memory(byte* buffer, word32 sz, int flag,
     }
 
 #ifdef WOLFSSL_DEBUG_MEMORY
-    printf("Allocated %d bytes for static memory @ %p\n", ava, pt);
+    fprintf(stderr, "Allocated %d bytes for static memory @ %p\n", ava, pt);
```

Comment:
some of these debug messages in memory.c I think are ok going to stdout?

Reply:
changed those for consistency -- a ton of them were already set up to go to stderr.  so either these go to stderr, or those go to stdout.  `grep` memory.c for printf and see what you think.